main.pm

```diff
@@ -339,8 +339,11 @@ sub is_reboot_after_installation_necessary() {
 
 sub load_inst_tests() {
     loadtest "installation/welcome.pm";
-    if (!check_var('BACKEND', 'ipmi')) { # network installs in general, but we have no setting for it yet
-      loadtest "installation/check_medium.pm";
+    if (!check_var('BACKEND', 'ipmi') || !check_var('BACKEND', 's390x')) { # network installs in general, but we have no setting for it yet
```

Comment:
You dont know why you merged it, I dont know why I wrote it - will fix when near a computer